crypto/src/randomness.rs

```diff
@@ -13,6 +13,7 @@ pub enum DomainSeparationTag {
     WindowedPoStChallengeSeed = 3,
     SealRandomness = 4,
     InteractiveSealChallengeSeed = 5,
+    WindowPoStDeadlineAssignment = 6,
```

Comment:
if you're adding this type, maybe is best to update to most recent:
```
	DomainSeparationTag_TicketProduction DomainSeparationTag = 1 + iota
	DomainSeparationTag_ElectionProofProduction
	DomainSeparationTag_WinningPoStChallengeSeed
	DomainSeparationTag_WindowedPoStChallengeSeed
	DomainSeparationTag_SealRandomness
	DomainSeparationTag_InteractiveSealChallengeSeed
	DomainSeparationTag_WindowedPoStDeadlineAssignment
```
are options now so the codes are off (probably should only specify 1 for TicketProduction and let it autoincrement in this case)